frontend/src/metabase/reference/containers/FieldXray.jsx

```diff
@@ -0,0 +1,156 @@
+/* @flow */
+import React, { Component } from 'react'
+import Visualization from 'metabase/visualizations/components/Visualization'
+
+import { connect } from 'react-redux'
+
+import type { Field } from 'metabase/meta/types/Field'
+
+type Props = {
+    fetchXrayData: () => void,
+    field: Field,
+    xray: {}
+}
+
+class FieldXray extends Component {
+    props: Props
+
+    componentWillMount() {
+        this.props.fetchXrayData(this.props.field.id)
+    }
+    render () {
+        return (
+            <div style={{
+                maxWidth: 550,
+                marginLeft: 'calc(48px + 1rem)',
+            }}>
+                <h3>XRAY</h3>
+                <div className="Grid Grid--1of2 Grid--gutters mt1">
+                    <div className="Grid-cell">
+                        <div className="bordered rounded shadowed py2">
+                            <Visualization
+                                series={[this.props.xray.min]}
+                            />
+                        </div>
+                    </div>
+                    <div className="Grid-cell">
+                        <div className="bordered rounded shadowed py2">
+                            <Visualization
+                                series={[this.props.xray.max]}
+                            />
+                        </div>
+                    </div>
+                </div>
+                <Visualization
```

Comment:
I could go either way. The visualizations have a lot of logic that helps charts look good for different shapes of data, sizes, etc, whereas I'm assuming X-rays will be much more limited.

But if we want things like tooltips then I think re-using the existing visualizations make sense. I'd probably just make some wrapper components like `XrayHistogram` that take the simple x-ray data and turn it into what Visualization expects.

Actually I'd make those classes either way, then we can easily swap out the implementation. Start with whatever implementation is easiest and go from there.